server/constants/constants.go

```diff
@@ -92,15 +92,16 @@ const (
 	QueryParamPerPage     = "per_page"
 
 	// Filters
-	FilterCreatedByMe          = "me"
-	FilterCreatedByAnyone      = "anyone"
-	FilterAllChannels          = "all_channels"
-	FilterAll                  = "all"
-	FilterBoards               = "boards"
-	FilterRepos                = "repos"
-	EventTypeRelease           = "release"
-	FilterReleaseDefinitionID  = "releaseDefinitionId"
-	FilterReleaseEnvironmentID = "releaseEnvironmentId"
+	FilterCreatedByMe            = "me"
+	FilterCreatedByAnyone        = "anyone"
+	FilterAllChannels            = "all_channels"
+	FilterAll                    = "all"
+	FilterBoards                 = "boards"
+	FilterRepos                  = "repos"
+	FilterPipelines              = "pipelines"
+	EventTypeRelease             = "release"
```

Comment:
don't we need any prefix of `Filter` here?

Reply:
No, its not a filter. Separated it out to avoid the confusion